### ISSUE TITLE
Startup errors

### ISSUE THREAD

**Issue by FlyingDiver** [FlyingDiver/Indigo-BondHome#4]
Errors in log when client devices started before bridge device:
```
   Bond Home                       Starting BondHome
   Bond Home                       All Shutters (73e68b9d): Starting bondDevice Device 1183074075
   Bond Home Error                 exception in deviceStartComm(All Shutters (73e68b9d)): u'ZZBL41370'
   Bond Home                       Fireplace (6296bd73): Starting bondDevice Device 468285124
   Bond Home Error                 exception in deviceStartComm(Fireplace (6296bd73)): u'ZZBL41370'
   Bond Home                       Great Room Bond: Starting bondBridge Device 596787479
```

**Comment by FlyingDiver** [comment]
Happens when client devices start before the associated bridge device.  Need to rework device startup code.

**Comment by FlyingDiver** [comment]
Should be fixed in 0.2.0.
